static/src/components/site-header/site-header.ts

```diff
@@ -101,6 +103,50 @@ export class SiteHeader {
         });
     }
 
+    protected handleArrowDown() {
+        window.addEventListener("keyup", (event) => {
+            const key = event.key || event.keyCode;
+            if (key === "ArrowDown" || key === 40) {
+                const parent = ( <HTMLElement>event.target ).parentElement;
+                if (parent) {
+                    const nextElement = parent.nextElementSibling;
+                    if (nextElement) {
+                        const innerElement = nextElement.querySelector('.site-header__nav > ul > li ul li > a') as HTMLElement;
+                        if (this.navExpanded && !this.element.contains(innerElement as HTMLElement)) {
+                            this.navExpanded = false;
+                            this.toggleNavVisibility();
+                        }
+                        else {
+                            innerElement.focus();
+                        }
+                    }
+                }
+            }
+        });
+    }
+
+    protected handleArrowUp() {
+        window.addEventListener("keyup", (event) => {
+            const key = event.key || event.keyCode;
+            if (key === "ArrowUp" || key === 38) {
+                const parent = ( <HTMLElement>event.target ).parentElement;
+                if (parent) {
+                    const previousElement = parent.previousElementSibling;
+                    if (previousElement) {
+                        const innerElement = previousElement.querySelector('a');
```

Comment:
I think this is wrong.  The `!(this.element.contains(innerElement)` will always return true right?  Every `li` has an anchor in it?  

Based on feedback from Auromita, I think this check can go away.  She stated that she didn't want the element to close via the [arrow keys](https://umd-digital.slack.com/archives/G016K8GCG5N/p1624385530033100?thread_ts=1624385467.032300&cid=G016K8GCG5N).  

Even though the behavior is working, the code can be cleaned up. 

Notes: 

If you need to declare the type (example HTMLElement), you should do so when the element is instantiated. 

For the event, it would be something like `event: KeyboardEvent`. 
For an element like an anchor, it would be `HTMLAnchorElement`.

If you don't know what the element is going to be, cast it to HTMLElement. 

If you don't know what the event is going to be, cast it to InputEvent.